frontend/src/app/pages/exercises/exercise/shared/utility/parse-csv.ts

```diff
@@ -0,0 +1,296 @@
+import type {
+    ConditionParameters,
+    ImageProperties,
+    PatientStatus,
+} from 'digital-fuesim-manv-shared';
+import {
+    Breathing,
+    Circulation,
+    Disability,
+    Exposure,
+    PatientHealthState,
+    PretriageInformation,
+    BiometricInformation,
+    PatientTemplate,
+} from 'digital-fuesim-manv-shared';
+
+export interface PatientData {
+    code: string;
+    templates: PatientTemplate[];
+}
+const defaultImage: ImageProperties = {
+    url: '/assets/male-patient.svg',
+    height: 80,
+    aspectRatio: 1,
+};
+
+export function parsePatientData(importString: string) {
+    const patientData: PatientData[] = [];
+
+    const splitString = importString.split('|');
+    const columns = 106;
+    // first 106 entries are the headers
+    for (let i = columns; i + columns - 1 < splitString.length; i += columns) {
+        // 0, 1, 2 are orginasational data for anaolg training
+        // 3 and 4 are personalInformation that will be randomly generated
+        const biometricInformation = BiometricInformation.create(
+            splitString[i + 7]!,
+            Number(splitString[i + 5]),
+            splitString[i + 6] === 'M'
+                ? 'male'
+                : splitString[i + 6] === 'F'
```

Comment:
```suggestion
                : splitString[i + 6] === 'W'
```

Or something like this; currently, all patients are either male or undefined.